spyder/plugins/remoteclient/tests/test_files.py

```diff
@@ -96,7 +95,7 @@ async def test_copy_file(
             ) == {"success": True}
 
         async with file_api_class() as file_api:
-            ls_content = await file_api.ls(self.remote_temp_dir)
+            ls_content = [ls_file async for ls_file in file_api.ls(self.remote_temp_dir)]
```

Comment:
```suggestion
            ls_content = [
                ls_file async for ls_file in file_api.ls(self.remote_temp_dir)
            ]
```

This line is too long too.